BedrockServer.cpp

```diff
@@ -1254,7 +1249,8 @@ void BedrockServer::_resetServer() {
     atomic_store(&_syncNode, shared_ptr<SQLiteNode>(nullptr));
     _shutdownState = RUNNING;
     _shouldBackup = false;
-    _commandPort = nullptr;
+    _commandPortPublic = nullptr;
```

Comment:
Breaking how?

Reply:
like if we didn't also change auth at the time as we deploy these changes auth wouldn't start back up right? and so also anyone who updates bedrock needs to change whatever plugins they have to pass these 2 new arguments otherwise they won't work?